### PR TITLE
derotation working

### PR DESCRIPTION
**Pull request by rmeertens**
Description: bebop image can compensate for pitch with this module....
Note that this pull request depends on the discussion in: 
https://github.com/paparazzi/paparazzi/pull/1808
https://github.com/paparazzi/paparazzi/pull/1807
Until these are fixed this cool module can't be used :(

### CODE REVIEW

File: sw/airborne/boards/bebop/mt9f002.c [hunk 1]
```diff
@@ -781,6 +781,13 @@ static void mt9f002_set_gains(struct mt9f002_t *mt)
   write_reg(mt, MT9F002_GREEN2_GAIN, mt9f002_calc_gain(mt->gain_green2), 2);
 }
 
+void set_offset(struct mt9f002_t *mt){
+	 write_reg(mt, MT9F002_X_ADDR_START, mt->offset_x , 2);
```
Comment (flixr) [inline sw/airborne/boards/bebop/mt9f002.c]: please fix indentation...

File: sw/airborne/boards/bebop/mt9f002.h [hunk 2]
```diff
@@ -77,6 +77,8 @@ struct mt9f002_t {
   struct i2c_transaction i2c_trans;   ///< I2C transaction for comminication with CMOS chip
 };
 
+
+extern void set_offset(struct mt9f002_t *mt);
```
Comment (flixr) [inline sw/airborne/boards/bebop/mt9f002.h]: doxygen description of what this function actually does would be nice...
Also would make sense to call it mt9f002_set_offset

File: sw/airborne/boards/bebop/board.c [hunk 3]
```diff
@@ -33,6 +33,7 @@
 #include "mt9v117.h"
 #include "mt9f002.h"
 #include "mcu.h"
+#include BOARD_CONFIG
```
Comment (flixr) [inline sw/airborne/boards/bebop/board.c]: why include BOARD_CONFIG here? wouldn't it be sufficient to include boards/bebop.h?

**Comment by flixr** [comment]
is this obsolete now with #1855 ?

**Comment by wilcoschoneveld** [comment]
yes this can be closed

**Pull request closed without merging**